pkgs/development/python-modules/onnx2pytorch/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  torch,
+  onnx,
+  onnxruntime,
+  torchvision,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "onnx2pytorch";
+  version = "0.4.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Talmaj";
+    repo = "onnx2pytorch";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-rEoFh96/yAmfGKJ/7pz4YT3pgee+CrpZcbt5Yl/IlbU=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    torch
+    onnx
+    onnxruntime
+    torchvision
+  ];
+
+  pythonImportsCheck = [ "onnx2pytorch" ];
+
+  pythonRelaxDeps = [ "torchvision" ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  disabledTests = [
+    "test_loop_sum" # module 'numpy' has no attribute 'bool'
```

Comment:
This package is not maintained, but does it actually work?
For example, this error comes from numpy 1.24, released over a year and a half ago, and it would be risky to disable it without a patch or similar.

https://numpy.org/doc/stable/release/1.24.0-notes.html#expired-deprecations